scripts/IntentData.py

```diff
@@ -76,23 +76,27 @@ def addRawOutput(self, rawOutputs, labelsMap):
         """ Read the raw output and store all data from it - 
             channel outputs, context variables and jumpto definitions. """
         self._rawOutputs.append(rawOutputs)
-        if not isinstance(rawOutputs, tuple) or len(rawOutputs) < 1:
+        if not isinstance(rawOutputs, tuple) or sum([1 if x else 0 for x in rawOutputs]) == 0:
```

Comment:
This checks for an empty output. Changed from length to scan of all items in array. An empty output cal also be an array of 3 Nones.

Reply:
might be worth to add a comment in the code   #check for empty rawOutputs

Reply:
Comment added.